backend/api/python_client_template/setup.mustache

```diff
@@ -0,0 +1,51 @@
+# coding: utf-8
+
+{{>partial_header}}
+
+from setuptools import setup, find_packages  # noqa: H301
+
+NAME = "{{{projectName}}}"
+VERSION = "{{packageVersion}}"
+{{#apiInfo}}
+{{#apis}}
+{{^hasMore}}
+# To install the library, run the following
+#
+# python setup.py install
+#
+# prerequisite: setuptools
+# http://pypi.python.org/pypi/setuptools
+
+REQUIRES = [
+    "certifi>=2017.4.17",
+    "python-dateutil>=2.1",
+    "six>=1.10",
+    "urllib3>=1.23"
+]
+
+{{#asyncio}}
+REQUIRES.append("aiohttp")
+{{/asyncio}}
+{{#tornado}}
+REQUIRES.append("tornado")
+{{/tornado}}
+
+setup(
+    name=NAME,
+    version=VERSION,
+    description="{{appName}}",
+    author="google",
+    author_email="{{infoEmail}}",
+    license="Apache 2.0",
```

Comment:
@rmgogogo I hardcoded the license info here, because in the original template provided by swagger, it was not generated.